.github/workflows/deploy-oss-catalog.yml

```diff
@@ -35,3 +35,14 @@ jobs:
           gcs_bucket_name="prod-airbyte-cloud-connector-metadata-service"
           catalog_path="airbyte-config/init/src/main/resources/seed/oss_catalog.json"
           gsutil -h "Cache-Control:public, max-age=10" cp "$catalog_path" "gs://$gcs_bucket_name/oss_catalog.json"
+      - name: Check PAT rate limits
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.OCTAVIA_4_ROOT_ACCESS }} \
+            ${{ secrets.OCTAVIA_PAT }}
+      - name: Trigger Cloud catalog generation
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ env.PAT }}
+          repository: airbytehq/airbyte-cloud
+          event-type: generate-cloud-catalog        
```

Comment:
Oh! this API is much cleaner than it used to be when we had to curl some random api.github.com URL!

Reply:
I didn't know `respository-dispatch` even existed, guess I've never had to trigger workflows across repos. Cool 😄